code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -35,5 +36,13 @@ Bonus
 				M.emote("sniff")
 			else
 				M.emote("sneeze")
-				A.spread(A.holder, 5)
-	return
\ No newline at end of file
+				Spreadzone(M, A)
+				var/obj/item/I = M.get_active_hand()
+				if(I && I.w_class > 0)

```

Comment:
I'm really not sure. I'll redo it for tonight. I think it's to check if anything is even being held in the first place.